dt-contacts/base-setup.php

```diff
@@ -534,6 +538,9 @@ public static function dt_user_list_filters( $filters, $post_type ){
                 ],
                 'query' => [
                     'sort' => '-post_date',
+                    'overall_status' => [
```

Comment:
@squigglybob Does this one do anything now?

Reply:
yes it does. Having the query stuff directly under the query object still works along side the nested fields object. The code for checking for -closed also still checks in both places too.